common/service/dynamicconfig/constants.go

```diff
@@ -827,6 +830,12 @@ const (
 	WorkerThrottledLogRPS
 	// ScannerPersistenceMaxQPS is the maximum rate of persistence calls from worker.Scanner
 	ScannerPersistenceMaxQPS
+	// ScannerMaxOrphanTasks is the maximum number of orphans to delete in one batch
```

Comment:
Can you open another PR in https://github.com/uber/cadence-docs to add the new configuration in https://cadenceworkflow.io/docs/operation-guide/setup/#sysworker-dynamic-configuration ? 

This is a new process we started recently to help our document updated. Otherwise it's super difficult to maintain our documentation.

Reply:
https://github.com/uber/cadence-docs/pull/29

Reply:
thanks!

But also sorry, you may have to update the PR after addressing comments about renames